tmva/tmva/src/DNN/Architectures/Cpu/Dropout.hxx

```diff
@@ -23,8 +23,8 @@ namespace DNN  {
 template<typename AFloat>
 void TCpu<AFloat>::DropoutForward(TCpuTensor<AFloat> & A, 
                                   TDescriptors * /*descriptors*/,
-                                  TWorkspace   * /*workspace*/, 
-                                  AFloat dropoutProbability)
+                                  TWorkspace   * /*workspace*/,
+                                  Scalar_t dropoutProbability)
```

Comment:
not sure if this change is needed

Reply:
Was motivated by:

```
/home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/tmva/src/DNN/Architectures/Cpu/Dropout.hxx:24: warning: no uniquely matching class member found for 
  template < AFloat >
  void TMVA::DNN::TCpu< AFloat >::DropoutForward(TMVA::DNN::TCpuTensor< AFloat > &A, TDescriptors *, TWorkspace *, AFloat dropoutProbability)
Possible candidates:
  'static void TMVA::DNN::TCpu< AReal >::DropoutForward(Tensor_t &A, TDescriptors *descriptors, TWorkspace *workspace, Scalar_t p)' at line 448 of file /home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/tmva/inc/TMVA/DNN/Architectures/Cpu.h
  'static void TMVA::DNN::TCpu< AReal >::DropoutForward(Matrix_t &A, Scalar_t p)' at line 453 of file /home/sftnight/build/workspace/root-makedoc-master/rootspi/rdoc/src/master/tmva/tmva/inc/TMVA/DNN/Architectures/Cpu.h
```

Reply:
It's not needed, but still better to be consistent. I would just suggest `TCpu<AFloat>::Scalar_t` for more information to the reader.